### DISCUSSION TITLE
ScreenToGif 2.38.1

### DISCUSSION THREAD

**Discussion by NickeManarin** [NickeManarin/ScreenToGif#1208] [Announcements]
# What's new?

• Just bug fixes and translation updates.  

### Fixed:

♦ Alt + Arrow Left/Right commands were not working.  
♦ Exporting as video with FFmpeg resulted in the last frame timing being ignored.  
♦ Some localizable text was not being loaded.  

### More info:

[Known issues](https://github.com/NickeManarin/ScreenToGif/wiki/Help-▪-Known-issues)

<hr /><em>This discussion was created from the release <a href='https://github.com/NickeManarin/ScreenToGif/releases/tag/2.38.1'>ScreenToGif 2.38.1</a>.</em>

**Comment by letifrank** [comment]
Hi, how are u? I'm trying to open a mkv file to convert into a gif and ScreenToGif keeps crashing. 
I select the file, but it keeps loading and crashes after. Do you have any idea what it might be? Thank you!!

![Screenshot_1](https://github.com/NickeManarin/ScreenToGif/assets/127629580/cbc3ba5f-4061-4241-9a5a-7c1d92faa078)

**Reply by NickeManarin** [reply]
Hello, I'm good and you? 
You may need to select FFmpeg at the top right.
But first you need to make sure that you have FFmpeg, head over the Options window > Extras and download FFmpeg from there.
